security/import_certificates_match.mo.md

```diff
@@ -0,0 +1,49 @@
+# [MO] Import certificates into Match *(~30 min)*
+
+## Owner: [Louis Lagrange](https://github.com/Minishlink)
+
+When using Fastlane's [Match](https://github.com/fastlane/fastlane/tree/master/match), you might want to update the certificates
+or provisioning profiles manually if you don't have enough rights on the publisher's Apple developer account.
```

Comment:
You can explicit what "don't have enough right" means: "if you don't have enough rights (meaning that you're unable to create certificates and provisioning by your own or by using match)".